span/daq.py

```diff
@@ -4,33 +4,37 @@
 from time import sleep
 
 class MyDAQ:
+    __samplerate: int
+    __name: str
+    
     def __init__(self):
-        self.__samplerate = None
-        self.__name = None
+        pass
     
     @property
-    def samplerate(self):
+    def samplerate(self) -> int:
         return self.__samplerate
     
     @samplerate.setter
-    def samplerate(self, newSamplerate):
-        assert newSamplerate > 0 and type(newSamplerate) == int, "Samplerate should be a positive integer!"
+    def samplerate(self, newSamplerate: int) -> None:
+        assert isinstance(newSamplerate, int), "Samplerate should be an integer."
+        assert newSamplerate > 0 , "Samplerate should be positive."
+        
         self.__samplerate = newSamplerate
     
     @property
-    def name(self):
+    def name(self) -> str:
         return self.__name
     
     @name.setter
-    def name(self, newName):
-        assert type(newName) == str, "Name should be a string!"
+    def name(self, newName: str) -> None:
+        assert isinstance(newName, str), "Name should be a string."
         self.__name = newName
     
-    def _addOutputChannels(self, task, channels):
+    def _addOutputChannels(self, task: dx.task.Task, channels) -> None:

```

Comment:
`channels` can be either `str` or `list` of `str`, so type hinting would be `str | list[str]`, or `Union[str, list[str]]`, depending on Python version